index.js

```diff
@@ -124,11 +124,15 @@ Collector.prototype.flush = function(callback) {
 // @param [Object] value the data
 // @param [Object] tags the tags (optional)
 Collector.prototype.collect = function(seriesName, value, tags) {
```

Comment:
Sure. I like `fields` because it implies that we can pass in more than one field, which encompasses the ability to pass in only one field.